bigquery/datasets.rb

```diff
@@ -41,10 +39,9 @@ def list_datasets project_id:
   # [START list_datasets]
   # project_id = "Your Google Cloud project ID"
 
-  require "google/cloud"
+  require "google/cloud/bigquery"
 
-  gcloud   = Google::Cloud.new project_id
-  bigquery = gcloud.bigquery
+  bigquery = Google::Cloud::Bigquery.new
```

Comment:
project: project_id is necessary because this is inside a function which has a project_id parameter.